src/components/ImageSlider/ImageSlider.stories.tsx

```diff
@@ -0,0 +1,58 @@
+// eslint-disable-next-line react/jsx-props-no-spreading
+import React, { useState } from 'react';
+import type { Story, Meta } from '@storybook/react';
+import ImageSlider, { ImageSliderProps, ImagesInterface } from './ImageSlider';
+
+export default {
+  title: 'Components/ImageSlider',
+  component: ImageSlider,
+} as Meta;
+
+const images: ImagesInterface[] = [
+  {
+    id: 'first image',
+    image:
+      'https://hips.hearstapps.com/hmg-prod.s3.amazonaws.com/images/golden-retriever-royalty-free-image-506756303-1560962726.jpg?crop=0.672xw:1.00xh;0.166xw,0&resize=640:*',
```

Comment:
We shouldn't depend on external images, put some images in the public folder and use relative import to show them.